### PR TITLE
add-aws-resolver

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/roadmap/issues/1249

### CODE REVIEW

File: helm/dns-operator-aws/templates/deployment.yaml [hunk 1]
```diff
@@ -41,13 +41,14 @@ spec:
...
+        - --associate-resolver-rules={{ .Values.associateResolveRules }}
         resources:
           requests:
             cpu: 100m
-            memory: 50Mi
+            memory: 100Mi
           limits:
             cpu: 100m
-            memory: 80Mi
+            memory: 150Mi
```
Comment (AverageMarcus) [inline helm/dns-operator-aws/templates/deployment.yaml:51]: Not needed for this PR but do we maybe need a VPA for this app?

**Review by AverageMarcus — APPROVED** [review]
LGTM 👍

File: helm/dns-operator-aws/templates/deployment.yaml (hunk 1 above)
Comment (calvix) [reply helm/dns-operator-aws/templates/deployment.yaml:51]: yeah make sense, will drop a note

Comment (calvix) [reply helm/dns-operator-aws/templates/deployment.yaml:51]: there is actually a VPA, i guess I don't want this to get killed every time it want to boot :p

**Pull request merged**